tests/integration/dhcp_test.py

```diff
@@ -204,18 +560,74 @@ def _remove_veth_pair():
 def _setup_dhcp_nics():
     assert libcmd.exec_cmd(['ip', 'link', 'set', DHCP_SRV_NIC, 'up'])[0] == 0
     assert libcmd.exec_cmd(['ip', 'link', 'set', DHCP_CLI_NIC, 'up'])[0] == 0
-    assert libcmd.exec_cmd(['ip', 'addr', 'add', IPV4_ADDRESS1, 'dev',
-                            DHCP_SRV_NIC])[0] == 0
+    assert libcmd.exec_cmd(['ip', 'addr', 'add', "{}/24".format(DHCP_SRV_IP4),
+                            'dev', DHCP_SRV_NIC])[0] == 0
+    with open(SYSFS_DISABLE_RA_SRV, 'w') as fd:
+        fd.write('0')
+
+    # This stop dhcp server NIC get another IPv6 address from radvd.
     with open(SYSFS_DISABLE_IPV6_FILE, 'w') as fd:
         fd.write('0')
-    assert libcmd.exec_cmd(['ip', 'addr', 'add', IPV6_ADDRESS1, 'dev',
-                            DHCP_SRV_NIC])[0] == 0
+
+    assert libcmd.exec_cmd(['ip', 'addr', 'add', "{}/64".format(DHCP_SRV_IP6),
+                            'dev', DHCP_SRV_NIC])[0] == 0
 
 
 def _clean_up():
     libcmd.exec_cmd(['systemctl', 'stop', 'dnsmasq'])
+    libcmd.exec_cmd(['systemctl', 'stop', 'radvd'])
     _remove_veth_pair()
     try:
         os.unlink(DNSMASQ_CONF_PATH)
     except FileNotFoundError:
         pass
+    except OSError:
+        pass
+    try:
+        os.unlink(RADVD_CONF_PATH)
+    except FileNotFoundError:
+        pass
+    except OSError:
+        pass
+
+
+def _get_dns_conf():
+    """
+    Return a list of name server string configured in /etc/resolv.conf
```

Comment:
It is not using /etc/resolv.conf anymore but the constant. How about adding a keyword argument `resolv_conf=RESOLV_CONF_PATH` to document which resolv.conf it is using?

Reply:
Changed the comment to:

`Return a list of name server string configured in RESOLV_CONF_PATH.`